PROTO_tests/tests/series_test.py

```diff
@@ -41,6 +43,48 @@ def test_series_creation(self, dtype):
         with pytest.raises(ValueError):
             ak.Series(data=ak.arange(3), index=ak.arange(6))
 
+    @pytest.mark.parametrize("size", pytest.prob_size)
+    def test_series_creation_pandas_series(self, size):
+        str_vals = ak.random_strings_uniform(9, 10, size)
+        cat_vals = ak.Categorical(str_vals)
+        num_vals = ak.arange(size) * -2
+        idx = ak.arange(size) * -1
+
+        pd_str = pd.Series(str_vals.to_ndarray(), idx.to_ndarray())
+        pd_cat = pd.Series(cat_vals.to_pandas(), idx.to_ndarray())
+        pd_num = pd.Series(num_vals.to_ndarray(), idx.to_ndarray())
+
+        ak_str = Series(pd_str)
+        ak_cat = Series(pd_cat)
+        ak_num = Series(pd_num)
+
+        expected_str = Series(str_vals, index=idx)
+        expected_cat = Series(cat_vals, index=idx)
+        expected_num = Series(num_vals, index=idx)
+
+        ak_assert_series_equal(ak_str, expected_str)
+        ak_assert_series_equal(ak_cat, expected_cat)
+        ak_assert_series_equal(ak_num, expected_num)
```

Comment:
this doesn't matter at all, but we could reduce some code duplication:
```python
  vals = [ak.random_strings_uniform(9, 10, size), ak.Categorical(str_vals), ak.arange(size) * -2]
  for val in vals:
    idx = ak.arange(size) * -1
    pd_ser = pd.Series(val.to_ndarray(), idx.to_ndarray())
    ak_ser = Series(pd_ser)
    expected = Series(val, index=idx)
    ak_assert_series_equal(ak_ser, expected)
```

Reply:
I had to make some adjustments so it would run, but I did make the change.  Thanks.